EdFi.Ods.Utilities.Migration/Scripts/02 Upgrade/v34_to_v50/01 Structure/1010-AddNewDescriptorValuesForV50.sql

```diff
@@ -0,0 +1,191 @@
+VALUES ('uri://ed-fi.org/QuestionFormDescriptor', 'Radio box', 'Radio box', 'Radio box');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/QuestionFormDescriptor', 'Matrix of dropdowns', 'Matrix of dropdowns', 'Matrix of dropdowns');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/QuestionFormDescriptor', 'Ranking', 'Ranking', 'Ranking');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/QuestionFormDescriptor', 'Textbox', 'Textbox', 'Textbox');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/QuestionFormDescriptor', 'Matrix of textboxes', 'Matrix of textboxes', 'Matrix of textboxes');
+
+INSERT INTO edfi.QuestionFormDescriptor
+SELECT DescriptorId FROM edfi.Descriptor
+WHERE [Namespace] = 'uri://ed-fi.org/QuestionFormDescriptor';
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SourceSystemDescriptor', 'District', 'District', 'District');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SourceSystemDescriptor', 'School', 'School', 'School');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SourceSystemDescriptor', 'State', 'State', 'State');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SourceSystemDescriptor', 'Federal', 'Federal', 'Federal');
+
+INSERT INTO edfi.SourceSystemDescriptor
+SELECT DescriptorId FROM edfi.Descriptor
+WHERE [Namespace] = 'uri://ed-fi.org/SourceSystemDescriptor';
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'Applicant', 'Applicant', 'Applicant');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'Administrator', 'Administrator', 'Administrator');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'District', 'District', 'District');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'Community', 'Community', 'Community');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'Other', 'Other', 'Other');
+
+INSERT INTO edfi.Descriptor ([Namespace], CodeValue, ShortDescription, [Description])
+VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'Exit', 'Exit', 'Exit');
```

Comment:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/blob/450a8995cc17f6d61f691755763ef4f9b74fdbf1/Descriptors/SurveyCategoryDescriptor.xml#L30
```suggestion
VALUES ('uri://ed-fi.org/SurveyCategoryDescriptor', 'Exit', 'Exit', 'Exiting staff');
```